store/sqlstore/store_test.go

```diff
@@ -95,6 +97,47 @@ func StoreTestWithSqlStore(t *testing.T, f func(*testing.T, store.Store, storete
 	}
 }
 
+func TestDBConnector(t *testing.T) {
+	testDrivers := []string{
+		model.DATABASE_DRIVER_POSTGRES,
+		model.DATABASE_DRIVER_MYSQL,
+	}
+
+	for _, dr := range testDrivers {
+		settings := makeSqlSettings(dr)
+
+		store := &SqlStore{
+			settings: settings,
+		}
+		connector, err := driver.NewConnector(*settings.DriverName, *settings.DataSource)
+		require.NoError(t, err)
+		db := sql.OpenDB(connector)
+
+		store.master = getDBMap(settings, db)
+		store.stores.post = newSqlPostStore(store, nil)
+		store.stores.channel = newSqlChannelStore(store, nil)
+		store.stores.team = newSqlTeamStore(store)
+		store.stores.thread = newSqlThreadStore(store)
+		store.stores.user = newSqlUserStore(store, nil)
+		store.stores.preference = newSqlPreferenceStore(store)
+		store.stores.bot = newSqlBotStore(store, nil)
+		store.stores.scheme = newSqlSchemeStore(store)
+		err = store.GetMaster().CreateTablesIfNotExists()
+		require.NoError(t, err)
+
+		store.stores.post.(*SqlPostStore).createIndexesIfNotExists()
+		store.stores.channel.(*SqlChannelStore).createIndexesIfNotExists()
+
+		t.Run(dr, func(t *testing.T) {
+			// Just testing post store for now.
+			// This will eventually go away when it is replaced with RPC.
+			storetest.TestPostStore(t, store, store)
```

Comment:
I have already tested the entire store and everything passes. But I didn't want to slow down CI further. And it will change slightly when we replace via RPC, so kept it minimal for now.